src/components/ConversationSettings/DangerZone.vue

```diff
@@ -51,7 +51,7 @@
 					:open.sync="isDeleteConversationDialogOpen"
 					:name="t('spreed','Delete Conversation')"
 					:message="deleteConversationDialogMessage"
-					:container="container">
+					container="#settings-section_dangerzone">
```

Comment:
It's not critical, but I would recommend to use our own classes and not depend on upstream ids.

Reply:
Fair